pom.xml

```diff
@@ -53,10 +53,11 @@ under the License.
     </scm>
 
     <properties>
+        <!--    TODO: Remove AWS SDK v1 -->
         <aws.sdkv1.version>1.12.439</aws.sdkv1.version>
         <aws.sdkv2.version>2.26.19</aws.sdkv2.version>
```

Comment:
Suggestion: We can bump sdkv2 version to the latest as well

Reply:
Let's decouple the changes -we can always do that in a followup PR